Formula/mkectl.rb

```diff
@@ -0,0 +1,37 @@
+require "yaml"
+
+class Mkectl < Formula
+  desc "Tool for creating and managing MKE 4.x kubernetes clusters"
+  homepage "https://mirantis.github.io/mke-docs/"
+  VERSION = "v4.0.0"
```

Comment:
Instead of defaulting to a valid version, can we default to an invalid version so that it fails instead of passing because it tested an old version?

Reply:
@nwneisen I'm not sure what you mean here, this is the current GA version that will be updated to 4.0.1 once that is released. We can also have multiple copies of formulas if (or when) we want multiple versions available via homebrew.

Reply:
Gotcha. I was thinking this was something that would pull the latest version. If it's manual then this makes sense.